health/handler.go

```diff
@@ -45,30 +50,52 @@ func NewChecksHandler(healthPath, readyPath string) Checker {
 func (ch *checksHandler) AddLiveness(name string, check Check) {
 	ch.lock.Lock()
 	defer ch.lock.Unlock()
+	ch.logger.WithFields(logrus.Fields{
+		"Liveness path":  ch.livenessPath,
+		"Readiness path": ch.readinessPath,
+		"Name":           name,
+	}).Info("adding liveness check")
```

Comment:
i would prefer this to be warn so they show up in production

Reply:
It's not a pseudo error though, isn't `info` the default log level?

Reply:
warn is the log level in production